src/wwwroot/app.js

```diff
@@ -264,6 +266,11 @@
     var pageOpenForAll = openForAllUrls.includes(currentPath);
     var pageRequireLogout = requireLogoutUrls.includes(currentPath);
     var pageRequireTemporalAuth = requireTemporalAuthUrls.includes(currentPath);
+    var tokenPermissons = auth.decodeToken();
```

Comment:
I think that _tokenPermissons_ could be considered part of _decodedToken_ I would not add more calls to _decodeToken()_ than actually exists.